renderer/index.css

```diff
@@ -674,6 +674,13 @@ body.drag .app::after {
   padding: 5px;
   margin: 0 auto;
 }
+.player-controls .rate {
```

Comment:
Looks good, but I recommend hiding the rate whenever it's 1 (ie, 99% of the time) and showing it only when it's `!== 1`